test/exercism/user_test.rb

```diff
@@ -103,6 +103,36 @@ def test_delete_team_memberships_with_user
     refute TeamMembership.exists?(team: other_team, user: bob, inviter: alice), 'Unconfirmed TeamMembership was deleted.'
   end
 
+  def test_qualifying_comment_adds_to_table
+    alice = User.create(username: 'alice')
+    fred = User.create(username: 'fred')
+    submission = Submission.create(user: alice, language: 'python', slug: 'one')
+    submission.reload
+    comment = Comment.create(user: fred, submission: submission, body: 'something')
+    fred.increment_five_a_day if comment.qualifying?
+    count = FiveADayCount.where(:user_id => fred.id).first
+    assert_equal 1, count.total
+  end
```

Comment:
This should be tested on the user, not on the comment. We only need to test that `fred.increment_five_a_day` works, not that it increments in the correct circumstance, since you have a test for the `comment.qualifying?` method.